app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
```

Comment:
The CommentController should have a `new` method which is missing in your case. If you want to find more on why you need `new` method, you can have a look [at this article](https://stackoverflow.com/questions/2472393/rails-new-vs-create).

Reply:
@muneebulrehman Hi I used a partial to display the comment form thus there is no use for a new method since the create method handles both instanciating and validating the values passed to the new object.

Reply:
Hey @DaveZag, it is a project requirement. And as you can see, there is a difference between the `new` and `create` methods. And it when you have a form for input it is always recommended to call the `new` method on that form which is automatically submit on the `create` method.

Reply:
@muneebulrehman Thanks for the explanations. I will add them🙂